src/java/com/zimbra/graphql/repositories/impl/ZXMLAccountRepository.java

```diff
@@ -107,24 +116,33 @@ public ZXMLAccountRepository(GetAccountInfo accountInfoHandler, EndSession endSe
     public AccountInfo accountInfoGet(RequestContext rctxt) throws ServiceException{
         final ZimbraSoapContext zsc = GQLAuthUtilities.getZimbraSoapContext(rctxt);
         final AccountInfo info = new AccountInfo();
-        final AccountSelector selector = new AccountSelector(AccountBy.id,
-            zsc.getAuthToken().getAccountId());
-        final GetAccountInfoRequest request = new GetAccountInfoRequest(selector);
+        final AccountSelector selector = new AccountSelector(AccountBy.id, zsc.getAuthToken().getAccountId());
+        final GetAccountInfoRequest accountInfoRequest = new GetAccountInfoRequest(selector);
+        final GetInfoRequest request = new GetInfoRequest();
+        final Element accountResponse = XMLDocumentUtilities.executeDocument(
```

Comment:
This works for now.

We may consider determining if the `communityUrl` is requested before doing the entire `getAccountInfo` request just for that. Or, perhaps adding it to `getInfo` and deprecating the other since it is seemingly the only thing missing?